src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -26,6 +26,14 @@
         (let [loc (.-location js/window)]
           (str (.-protocol loc) "//" (.-host loc) "/#profile/nih-username-token={token}")))))
 
+(defn get-fence-link-href []
+  (str "https://qa.dcf.planx-pla.net" #_(get @config/config "fenceUrlRoot")
+       "/user/oauth2/authorize?response_type=code&client_id=REPLACE_ME&redirect_uri="
+       (js/encodeURIComponent
+        (let [loc (.-location js/window)]
+          (str "https:" #_(.-protocol loc) "//" (.-host loc) "/fence-callback")))
+       "&scope=openid+google_credentials"))
```

Comment:
Can we put this param before the `redirect_uri`? I think it would be easier to read.